src/Constant.tsx

```diff
@@ -107,80 +68,80 @@ export const drawTools: TDrawTools = {
 
 export const getIndicatorsTree = (): TIndicatorsTree[] => [
     {
-        id: 'momentum',
+        category: 'momentum',
         name: t.translate('Momentum'),
         icon: IndicatorCatMomentumIcon,
         items: [
             {
-                id: 'Awesome',
-                name: t.translate('Awesome Oscillator'),
+                name: 'AwesomeOscillator',
```

Comment:
@balakrishna-deriv I noticed that indicators names here vary greatly: some are written in camelCase, some in snake_case, some in lower case, some in upper case. Is there any naming convention for these?

Reply:
These are the ids in flutter-charts. Yes, they are inconsistent but I don't want to change them in the flutter-chart.
I've renamed the `name` prop to `flutter-chart-id`.